serde_arrow/Cargo.toml

```diff
@@ -43,7 +43,7 @@ arrow-37 = ["dep:arrow-array-37", "dep:arrow-schema-37", "dep:arrow-data-37", "d
 [dependencies]
 bytemuck = { version = "1", default-features = false }
 # TODO: make optional, only required for str -> date conversions
-chrono = { version = "0.4", features = ["std"], default-features = false }
+chrono = { version = "0.4.35", features = ["std"], default-features = false }
```

Comment:
Any particular reason to bump the minimum required version?

Reply:
changed by acc. reverted